### PR TITLE
Shuffled log levels

### PR DESCRIPTION
**Pull request by rain-on**

### CODE REVIEW

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java [hunk 1]
```diff
@@ -155,21 +155,21 @@ public void roundExpired(final RoundExpiry expire) {
 
   @Override
   public void handleProposalPayload(final Proposal proposal) {
-    LOG.debug("Received a Proposal Payload.");
+    LOG.trace("Received a Proposal Payload.");
```
Comment (jframe) [inline consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java]: I think these receives would be useful for debugging

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java [hunk 2]
```diff
@@ -155,21 +155,21 @@ public void roundExpired(final RoundExpiry expire) {
...
     actionOrBufferMessage(
         proposal, currentRound::handleProposalMessage, RoundState::setProposedBlock);
   }
 
   @Override
   public void handlePreparePayload(final Prepare prepare) {
-    LOG.debug("Received a Prepare Payload.");
+    LOG.trace("Received a Prepare Payload.");
```
Comment (jframe) [inline consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java]: debug?

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java [hunk 3]
```diff
@@ -155,21 +155,21 @@ public void roundExpired(final RoundExpiry expire) {
...
     actionOrBufferMessage(
         prepare, currentRound::handlePrepareMessage, RoundState::addPrepareMessage);
   }
 
   @Override
   public void handleCommitPayload(final Commit commit) {
-    LOG.debug("Received a Commit Payload.");
+    LOG.trace("Received a Commit Payload.");
```
Comment (jframe) [inline consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java]: debug?

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftController.java [hunk 4]
```diff
@@ -158,8 +158,9 @@ private void handleMessage(final Message message) {
   public void handleNewBlockEvent(final NewChainHead newChainHead) {
     final BlockHeader newBlockHeader = newChainHead.getNewChainHeadHeader();
     final BlockHeader currentMiningParent = currentHeightManager.getParentBlockHeader();
+    LOG.debug("Handling New Chain head event, chain length = {}", currentMiningParent.getNumber());
```
Comment (jframe) [inline consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftController.java]: isn't this the chain height not chain length? elsewhere is logs I think block is just used to infer block number.

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java [hunk 5]
```diff
@@ -193,18 +193,21 @@ public void handleCommitPayload(final Commit commit) {
   @Override
   public void handleRoundChangePayload(final RoundChange message) {
     final ConsensusRoundIdentifier targetRound = message.getRoundIdentifier();
-    LOG.info("Received a RoundChange Payload for {}", targetRound.toString());
+    LOG.trace("Received a RoundChange Payload for {}", targetRound.toString());
```
[the remark this answers is not in the document]
Comment (rain-on) [reply consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java]: done.

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java (hunk 1 above)
Comment (rain-on) [reply consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java]: these are in debug in the IbftRound (along with message sends) - thus I figured these were largely just a pass through (and potentially do not require logging at all).

Comment (jframe) [reply consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java]: good point. there is no benefit in logging those at the height manager and in the ibft round. so happy for this to be trace and also for the other receives in the height manager.

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftController.java (hunk 4 above)
Comment (rain-on) [reply consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftController.java]: Updated to be chain height.

**Comment by rain-on** [comment]
> Also noticed that this is being logged at info level. Think this should be debug level
> 
> ```
> NewRoundMessageValidator | No round change messages have a preparedCertificate, any valid block may be proposed.
> ```
went with trace - this is a really normal thing to happen, its not even debug in my opinion.

**Pull request merged**